cli/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 
 _Released 3/26/2024 (PENDING)_
 
+**Bugfixes:**
+
+- Fixed a hang in the test runner where Cypress would run indefinitely in record mode when CDP disconnects during the middle of a test. Fixes [#29209].
```

Comment:
this should be un `13.7.2` which should be on `develop`. 
```suggestion
- Fixed a hang in the test runner where Cypress would run indefinitely in record mode when CDP disconnects during the middle of a test. Fixes [#29209](https://github.com/cypress-io/cypress/issues/29209).
```

Reply:
Doh. Went to fast. Good call. Fixed with the merge.